example/spec/parallel/orca/OrcaBackupAndRecoverySpec.js

```diff
@@ -117,7 +117,13 @@ describe('The S3 Ingest Granules workflow', () => {
 
     const inputPayloadJson = fs.readFileSync(inputPayloadFilename, 'utf8');
     // update test data filepaths
-    inputPayload = await setupTestGranuleForIngest(config.bucket, inputPayloadJson, granuleRegex, testSuffix, testDataFolder);
+    inputPayload = await setupTestGranuleForIngest(
+      config.bucket,
+      JSON.stringify({ ...JSON.parse(inputPayloadJson), pdr: undefined }),
```

Comment:
This removes a needless PDR from the input payload - since it's using a common payload fixture, this seemed preferable to creating another fixture.